src/Core/Infrastructure/Configuration/User/Api/PatchUser/PatchUserController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * Copyright 2005 - 2022 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Core\Infrastructure\Configuration\User\Api\PatchUser;
+
+use Centreon\Application\Controller\AbstractController;
+use Centreon\Domain\Contact\Contact;
+use Centreon\Domain\Log\LoggerTrait;
+use Core\Application\Configuration\User\UseCase\PatchUser\PatchUser;
+use Core\Application\Configuration\User\UseCase\PatchUser\PatchUserPresenterInterface;
+use Core\Application\Configuration\User\UseCase\PatchUser\PatchUserRequest;
+use Symfony\Component\HttpFoundation\Request;
+
+final class PatchUserController extends AbstractController
+{
+    use LoggerTrait;
+
+    /**
+     * @param Request $request
+     * @param mixed $user
+     * @param PatchUser $useCase
+     * @param PatchUserPresenterInterface $presenter
+     * @return object
+     */
```

Comment:
Simple re-order of the arguments to match method signature
```suggestion
    /**
     * @param Request $request
     * @param PatchUser $useCase
     * @param mixed $user
     * @param PatchUserPresenterInterface $presenter
     * @return object
     */
```